packages/field-base/src/input-props-mixin.d.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { InputAriaMixin } from './input-aria-mixin.js';
+import { ValidateMixin } from './validate-mixin.js';
+
+/**
+ * A mixin to add `<input>` element to the corresponding named slot.
```

Comment:
I don't think it does that? I can't see the mixin adding any inputs. Maybe it just need to be worded better.

Reply:
Good catch, fixed.